test/utils/session-helper.test.js

```diff
@@ -1,7 +1,31 @@
-import { expect, vi, it, describe } from 'vitest';
-import { createResourceTtlFile } from '../../src/utils/network/session-helper';
+import {
+  addMockResourceAclTo,
+  getThingAll,
+  mockSolidDatasetFrom,
+  mockThingFrom
+} from '@inrupt/solid-client';
+import { expect, vi, it, describe, beforeEach, afterEach } from 'vitest';
+import {
+  createResourceTtlFile,
+  getContainerUrl,
+  hasTTLFiles
+} from '../../src/utils/network/session-helper';
 import { INTERACTION_TYPES } from '../../src/constants';
 
+const mockPodUrl = 'https://pod.example.com/';
+const mockSolidDataset = addMockResourceAclTo(mockSolidDatasetFrom(mockPodUrl));
+let session = {};
+
+vi.mock('@inrupt/solid-client');
+
+vi.mock('@inrupt/solid-client', async () => {
```

Comment:
You can remove lines 21 - 27 if you merge in the changes I made in that small unit test PR. The case you're trying to handle here will be handled by line 19.

Reply:
I'll need a getThingAll which mocks getting a list of Things.